analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -119,35 +119,30 @@ class GoMod(
         stashDirectories(projectDir.resolve("vendor")).use { _ ->
             val moduleInfoForModuleName = getModuleInfos(projectDir)
             val graph = getModuleGraph(projectDir, moduleInfoForModuleName)
-            val projectId = moduleInfoForModuleName.getMainModuleId()
-            val packageIds = graph.nodes - projectId
-            val packages = packageIds.mapTo(mutableSetOf()) { moduleInfoForModuleName.getValue(it.name).toPackage() }
-            val projectVcs = processProjectVcs(projectDir)
+            val packages = graph.nodes.mapNotNullTo(mutableSetOf()) {
```

Comment:
Commit message: Maybe https://go.dev/ref/mod#go-mod-file-replace is a better reference for `[2]`?

Reply:
I preferred [2], because of this comment: https://github.com/golang/go/issues/40513#issuecomment-668134054. I would have loved to link it directly, but this get rejected by commitlint.